src/Bridge/Symfony/Bundle/DependencyInjection/Compiler/AnnotationFilterPass.php

```diff
@@ -77,6 +78,9 @@ private function createFilterDefinitions(\ReflectionClass $reflectionClass, Cont
             $definition->setAutowired(true);
 
             foreach ($arguments as $key => $value) {
+                if (null === $reflectionConstructor = $reflectionClass->getConstructor()) {
```

Comment:
`$reflectionClass` could be `null` here. Also, this check should be done before the `foreach` loop.